nautobot/dcim/tests/integration/test_device_bulk_operations.py

```diff
@@ -0,0 +1,31 @@
+import uuid
+
+from nautobot.core.testing.integration import (
+    BulkOperationsTestCases,
+)
+from nautobot.dcim.models import Device
+from nautobot.extras.tests.integration import create_test_device
+
+
+class DeviceBulkOperationsTestCase(BulkOperationsTestCases.BulkOperationsTestCase):
+    """
+    Test devices bulk edit / delete operations.
+    """
+
+    model_menu_path = ("Devices", "Devices")
+    model_base_viewname = "dcim:device"
+    model_edit_data = {"serial": "Test serial"}
+    model_filter_by = {"location": "Test Location 2"}
+    model_class = Device
+    model_plural = Device._meta.verbose_name_plural
```

Comment:
Can we add logic in the base class to use `self.model_class._meta.verbose_name_plural` if `model_plural` isn't explicitly set?